src/olmo_core/launch/beaker.py

```diff
@@ -84,6 +84,16 @@ class BeakerWekaBucket(Config):
     mount: str
 
 
+DEFAULT_SETUP_STEPS = (
+    'git clone "${REPO_URL}" .',
+    'git checkout "${GIT_REF}"',
+    "git submodule update --init --recursive",
+    "conda shell.bash activate base",
```

Comment:
Looks like we do expect conda. If so, can we just define `PRIVATE_REPO_SETUP_STEPS`, and then it will Just Work ™ ?

Reply:
But this is just the default because we know conda will be on the default images.

Reply:
Ok, fair enough.